pkg/ruler/ruler.go

```diff
@@ -48,9 +48,6 @@ type Config struct {
 	ClientTLSConfig tls.ClientConfig `yaml:"ruler_client"`
 	// How frequently to evaluate rules by default.
 	EvaluationInterval time.Duration `yaml:"evaluation_interval"`
-	// Delay the evaluation of all rules by a set interval to give a buffer
-	// to metric that haven't been forwarded to cortex yet.
-	EvaluationDelay time.Duration `yaml:"evaluation_delay_duration"`
 	// How frequently to poll for updated rules.
```

Comment:
We should keep this, but deprecate. We can remove it after two minor versions.

Reply:
Is there anything special I need to do to deprecated a config field?